crates/c-api/include/wasmtime/engine.h

```diff
@@ -13,6 +13,11 @@
 extern "C" {
 #endif
 
+/**
+ * \brief Create a new reference to the same underlying engine.
+ */
```

Comment:
Could the documentation be updated here to indicate that `wasm_engine_delete` must be called on the returned reference?